File: desktop/src/uk/co/developmentanddinosaurs/games/dinner/DesktopLauncher.java
package uk.co.developmentanddinosaurs.games.dinner;

import com.badlogic.gdx.backends.lwjgl3.Lwjgl3Application;
import com.badlogic.gdx.backends.lwjgl3.Lwjgl3ApplicationConfiguration;
import uk.co.developmentanddinosaurs.games.dinner.DinnerGame;

// Please note that on macOS your application needs to be started with the -XstartOnFirstThread JVM argument
public class DesktopLauncher {
	public static void main (String[] arg) {
		Lwjgl3ApplicationConfiguration config = new Lwjgl3ApplicationConfiguration();
		config.setForegroundFPS(60);
		config.setTitle("Dino Dinner Democracy");
		new Lwjgl3Application(new DinnerGame(), config);
	}
}

File: desktop/src/main/kotlin/uk/co/developmentanddinosaurs/games/dinner/DesktopLauncher.kt
/** Main entrypoint into the DesktopLauncher */
package uk.co.developmentanddinosaurs.games.dinner

import com.badlogic.gdx.backends.lwjgl3.Lwjgl3Application
import com.badlogic.gdx.backends.lwjgl3.Lwjgl3ApplicationConfiguration

/**
 * Entrypoint for the Desktop launcher. Simply runs a main function that creates the Dino Dinner
 * Democracy game and launches it using lwjgl3.
 */
fun main() {
  val config = Lwjgl3ApplicationConfiguration()
  config.setForegroundFPS(60)
  config.setTitle("Dino Dinner Democracy")
  config.setWindowedMode(1600, 900)
  config.setWindowIcon("sprites/meat.png")
  Lwjgl3Application(DinnerGame(), config)
}
